[debug] [2020-04-16T05:32:22.474Z] ----------------------------------------------------------------------
[debug] [2020-04-16T05:32:22.477Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Admin\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-16T05:32:22.478Z] CLI Version:   8.0.3
[debug] [2020-04-16T05:32:22.478Z] Platform:      win32
[debug] [2020-04-16T05:32:22.478Z] Node Version:  v13.12.0
[debug] [2020-04-16T05:32:22.479Z] Time:          Thu Apr 16 2020 12:32:22 GMT+0700 (Indochina Time)
[debug] [2020-04-16T05:32:22.480Z] ----------------------------------------------------------------------
[debug] [2020-04-16T05:32:22.480Z] 
[debug] [2020-04-16T05:32:22.493Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-16T05:32:22.521Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\Admin\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async GoogleAuth.getClient (C:\Users\Admin\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\Admin\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
